src/Language/PureScript/Bridge/Printer.hs

```diff
@@ -58,21 +59,36 @@ moduleToText m = T.unlines $
   "-- File auto generated by purescript-bridge! --"
   : "module " <> psModuleName m <> " where\n"
   : map importLineToText (Map.elems (psImportLines m))
-  ++ [ "\nimport Data.Generic (class Generic)\n\n" ]
+  ++ [ ""
+     , "import Prelude"
+     , "import Data.Generic (class Generic)"
+     , "import Data.Maybe"
+     , "import Data.Lens"
+     , ""
+     ]
```

Comment:
If the generated code does not generate warnings, this would definitely be better.